simple.js

```diff
@@ -1381,19 +1378,22 @@ define(function(require, exports, module) {
                 // set Python's version
                 settings.set("project/python/@version", "python3");
 
-                // set pylint's flags
-                settings.set(
-                    "project/python/@pylintFlags",
-                    "-d all -e E -e F --generated-members=app.jinja_env.* --ignored-classes=SQLAlchemy --load-plugins pylint50"
-                );
-
                 // set PYTHONPATH
                 settings.set(
                     "project/python/@path",
                     "/home/ubuntu/.local/lib/python3.4/site-packages" +
                     ":/usr/lib/python3/dist-packages" +
                     ":/usr/local/lib/python3.4/dist-packages"
                 );
+
+                // set pylint's flags
+                settings.set(
+                    "project/python/@pylintFlags",
+                    "-d all -e E -e F --generated-members=app.jinja_env.* --ignored-classes=SQL,SQLAlchemy --load-plugins pylint50"
```

Comment:
They can go system-wide. I just picked `.local` because that's the first on `PYTHONPATH` per https://github.com/cs50/harvard.cs50.simple/blob/master/simple.js#L1393.